src/content/docs/en/recipes/sharing-state.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Share State Between Astro Components
+description: Learn how to share state across Astro components — with Nano Stores.
+i18nReady: true
+type: recipe
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+:::tip
+  Using Framework components? See [how to share state between Islands](/en/core-concepts/sharing-state/)!
+:::
+
+When building an Astro website, you may need to share state across components. Astro recommend the use of [Nano Stores](https://github.com/nanostores/nanostores) for shared client storage.
+
+## Recipe
+
+1. Install Nano Stores:
+
+  <PackageManagerTabs>
+    <Fragment slot="npm">
+    ```shell
+    npm install nanostores
+    ```
+    </Fragment>
+    <Fragment slot="pnpm">
+    ```shell
+    pnpm install nanostores
+    ```
+    </Fragment>
+    <Fragment slot="yarn">
+    ```shell
+    yarn add nanostores
+    ```
+    </Fragment>
+  </PackageManagerTabs>
+
+2. Create a store. In this example, the store tracks whether a dialog is open or not:
+
+  ```ts title="src/store.js"
+  import { atom } from 'nanostores';
+
+  export const isOpen = atom(false);
+  ```
+
+3. Import and use the store in a [`script`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script) tag in your components:
```

Comment:
Personal opinion; I don't think that we need to link the HTML docs here. If our readers are trying to implement stores, they probably know what HTML / JS is, thus are familiar with the `script` tag.

What do you think?

```suggestion
3. Import and use the store in a `<script>` tag in your components:
```